### DISCUSSION TITLE
I want to redirect my USB flash drive, but I don't know how to type this command

### DISCUSSION THREAD

**Discussion by LLeext** [FreeRDP/FreeRDP#9337] [Q&A]
I try to use USB redirection function, but do not know how to use, use /usb:    , but it seems to have no effect, I may use the wrong, this is my USB disk hardware ID: `USB\VID_13FE&PID_6300`, can you tell me how to redirect my USB disk?

**Comment by akallabeth** [comment]
* that depends a lot on where you connect to. `gnome-remote-desktop`, `ogon` and `xrdp` do not support low level USB redirection
* Windows does filter certain device types and prevents their use with low level USB redirection (mass storage is one of these)

so, if you want to redirect a USB drive:
1. mount it on your local machine 
2. use `/drive` option to redirect that mount point (drive letter on windows)

**Reply by LLeext** [reply]
This is the command I typed`wfreerdp.exe /u:*** /p:*** /v:*** /drive:E,E:\ -clipboard` 
But it didn't seem to work. Was I wrong to order it? I run xrdp on centos

**Reply by akallabeth** [reply]
works mighty fine here with todays nightly (sdl-freerdp.exe and wfreerdp.exe) against a windows server.
you might want to check if you have that enabled with your `xrdp` setup.
